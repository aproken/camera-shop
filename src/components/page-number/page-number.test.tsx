import { render, RenderResult } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import PageNumber from './page-number';

describe('PageNumber component', () => {
  it('should render correctly', () => {
    const currentPage = 2;
    const page = 3;

    const { container }: RenderResult = render(
      <MemoryRouter>
        <PageNumber currentPage={currentPage} page={page} />
      </MemoryRouter>
    );

    expect(container).toMatchSnapshot();
  });
});
