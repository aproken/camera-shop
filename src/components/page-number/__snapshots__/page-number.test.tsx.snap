// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNumber component should render correctly 1`] = `
<div>
  <li
    class="pagination__item"
  >
    <button
      class="pagination__link"
    >
      3
    </button>
  </li>
</div>
`;
